sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ModelBridgeInternal.java

```diff
@@ -205,6 +211,16 @@ public static User getV2User(CosmosUserProperties cosmosUserProperties) {
         return CosmosUserProperties.getFromV2Results(results);
     }
 
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
+    public static ClientEncryptionKey getClientEncryptionPolicys(CosmosClientEncryptionKeyProperties cosmosClientEncryptionKeyProperties) {
```

Comment:
>Encryptio [](start = 47, length = 9)

Shouldn't this be getClientEncryptionKey/getClientEncryptionKeys rather than policy?

Reply:
Done